packages/server/database/types/MeetingRetrospective.ts

```diff
@@ -1,3 +1,4 @@
+import {AutogroupReflectionGroupType} from '../../graphql/types/AutogroupReflectionGroup'
```

Comment:
-1 Our database types should not depend on GraphQL types. When we change our API, the underlying data does not change automatically. I missed this one before and it's causing your typecheck issue.